src/common/SPELLS_PRIEST.js

```diff
@@ -431,6 +444,20 @@ export default {
     icon: 'inv_enchant_essencemagiclarge',
   },
 
+  // Holy Tier Sets
+  HOLY_PRIEST_T20_2SET_BONUS_BUFF: {
+    id: 242270,
+    name: 'Holy Priest T20 2P Bonus',
+    icon: 'spell_holy_guardianspirit',
+    coeff: 1000,
```

Comment:
I think this value should be in the module, not in SPELLS. Also 'coeff' is a really vague name for this field.

Reply:
I think it should remain here since it's a trait and it's value could be changed in a future patch (unlikely but still possible). Maybe change the variable name but keeping the value itself is good for future-proofing

Reply:
Think of how the T20 2P for Disc Priest was recently nerfed in the patch. Had this value been used in multiple spots (as it is with my code), all you would have to do is change this one value in SPELLS files. Covers multiple bases and allows for new contributors to easily modify the value.